lib/internal/Magento/Framework/Mview/View.php

```diff
@@ -219,21 +219,32 @@ public function subscribe()
     }
 
     /**
-     * Remove subscriptions
-     *
-     * @return ViewInterface
-     * @throws Exception
+     * @inheritDoc
      */
-    public function unsubscribe()
+    public function unsubscribe(bool $dropTable = true): ViewInterface
```

Comment:
Make sense to have `false` as default value to preserve BC

Reply:
Hi @kandy. I'm assuming that by 'BC' you mean "backwards compatibility."

If the parameter was required (ie, had no default value), it would break backwards compatibility. As the parameter has a default value, anything that calls this method without any arguments will continue to work. I therefore see this as being compatible with previous code, and not breaking backwards compatibility at a code level.

I think you're trying to ask for this method to behave the same as it does currently when called without any arguments. This parameter only exists so that it can be used in the recurring setup script where the intention is to recreate all triggers. If it weren't for that requirement, the parameter would not exist at all.

Can you provide a use-case where it would be useful to keep the changelog table after calling `mview->unsubscribe()`?